python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_cugraph.py

```diff
@@ -52,13 +52,21 @@ def adjacency_matrix():
     return df
 
 
+# TODO: Tracking Issue https://github.com/rapidsai/cudf/issues/17934
+@pytest.mark.skip(
```

Comment:
Can we `pytest.mark.xfail(raises=TypeError, reason=...)` so we don't accidentally, perpetually skip these tests?

Reply:
Thanks this was actually fixed in #17936, so I can remove these xfails